syft/frameworks/torch/tensors/interpreters/replicated_shared.py

```diff
@@ -178,8 +178,8 @@ def __public_linear_operation(self, plain_text, operator):
             shares_map[players[0]][1],
         )
         shares_map[players[-1]] = (
-            operator(shares_map[players[-1]][-1], remote_plain_text[-1]),
```

Comment:
P1 has share x1, x2; P2 has x2,x3 and P3 has x3 and x1 - when adding a constant we will have
P1 has share x1 + c, x2, P2 has share x2,x3 (nothing changed) and P3 has x3 and x1 + c